packages/fast-web-utilities/src/dom.ts

```diff
@@ -1,6 +1,28 @@
 import { canUseDOM } from "exenv-es6";
 import { isBoolean } from "lodash-es";
 
+/**
+ * A test that ensures that all arguments are HTML Elements
+ */
+export function isHTMLElement(...args: unknown[]): boolean {
```

Comment:
Is it possible to make this a type guard so the type system knows these are HTMLElements?

Reply:
I can update to `HTMLElement[]`